modules/using-a-pre-deployed-image-to-improve-efficiency.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/optimizing-compute-resources.adoc
+
+[id='using-a-pre-deployed-image-to-improve-efficiency-{context}']
+= Using a pre-deployed image to improve efficiency
+
+You can create a base {product-title} image with a number of tasks built-in to
+improve efficiency, maintain configuration consistency on all node hosts, and
+reduce repetitive tasks. This is known as a pre-deployed image. For example,
+because every node requires the `ose-pod` image in order to run pods, each node
+has to periodically connect to the container image registry in order to pull the
+latest image. This can become problematic when you have 100 nodes attempting
+this at the same time, and can lead to resource contention on the image
+registry, waste of network bandwidth, and increased pod launch times.
+
+.Procedure
+
+To build a pre-deployed image:
```

Comment:
@jeremyeder I feel like this procedure is a bit weak. What are specific steps for the user?

Reply:
It is a bit weak in terms of specifics, but it is accurate.  What I worry about is how RHCOS moves us away from this requirement, and we don't yet have clarity on what the bring-your-own-host (i.e. RHEL) path will look like.  It's possible that this section will still be beneficial.  But until we have 4.0/RHCOS on AWS test results in-hand, it is hard to say on whether we should keep this section.

Reply:
@jeremyeder Thanks! I made some updates. Do you think we should merge this as-is and follow up when we have more details, or do you prefer to keep this open for now?